x-pack/plugins/ml/common/types/anomaly_detection_jobs/datafeed.ts

```diff
@@ -15,4 +15,9 @@ export type ChunkingConfig = estypes.MlChunkingConfig;
 
 export type Aggregation = Record<string, estypes.AggregationsAggregationContainer>;
 
-export type IndicesOptions = estypes.MlDatafeedIndicesOptions;
+export interface IndicesOptions {
```

Comment:
the client doesn't contain this type anymore. @delvedor what is a proposed alternative?

Reply:
Heya, it's exported as `IndicesOptions` :)

Reply:
I checked the types started with `ML*` prefix 😄

Reply:
Sorry about that! We noticed that many apis were sharing that configuration, so we standardized it once and for all!